ORG-GOVERNANCE.md

```diff
@@ -0,0 +1,57 @@
+# Charter for the Fairlearn Organization
+
+This is the organizational charter for the Fairlearn Organization (the "Organization").
+By adding their name or the name of their Project to the [MEMBERS.md](./MEMBERS.md) file, the
+SC members, the AB members and the Project Maintainers agree as follows.
+
+## 1. Mission
+
+Provide a set of open source resources to assess and improve fairness of AI systems.
+
+## 2. Membership
+
+The Organization is composed of:
+
+**2.1 Projects**. Projects consist of one or more repositories. The Projects are represented and activities in each Project are organized by its Maintainers as set forth in the Project Governance Policy.
+
+**2.2 Steering Committee**. The Steering Committee (the "SC") is responsible for project selection, project oversight, policy oversight, and trademark management for the Organization.
+
+**2.3 Advisory Board**. The Advisory Board (the "AB") consists of experts that advise Projects and the SC. The members of the Advisory Board can be added or removed by the SC.
+
+## 3. Steering Committee
+
+**3.1 Composition**. The SC voting members are listed in the [MEMBERS.md](./MEMBERS.md) file in the repository.
+Voting members may be added or removed by 3/4 affirmative vote of the SC.
+The SC will appoint a Chair responsible for organizing SC activity.
+
+**3.2 Decision Making**. The SC will strive for all decisions to be made by consensus. If consensus cannot be reached, the Steering Committee will make the decision by a vote.
+
+**3.3 Voting**. The SC Chair will call a vote with reasonable notice to the SC, setting out a discussion period and a separate voting period. Any discussion may be conducted in person or electronically by text, voice, or video. The discussion will be open to the public. In any vote, each voting representative will have one vote. Except as specifically noted elsewhere in this Charter, decisions by vote require a simple majority vote of all voting members.
+
+**3.4 Termination of Membership**. The membership of a SC member will terminate if any of the following occur:
+
+**3.4.1 Resignation**. Written notice of resignation to the SC.
+
+**3.4.2 Unreachable Member**. If a member is unresponsive at their listed handle for more than one month the SC may vote to remove the member.
```

Comment:
happy to add, but I think it's redundant (since we have the "voting" section below).

Reply:
Actually, I think I'm going to update here, because as written it's somewhat ambiguous. I believe it should be 3/4 since it concerns adding / removing a member.